_sources/api/pyface.util.rst.txt

```diff
@@ -2,26 +2,22 @@ pyface.util package
 ===================
 
 .. automodule:: pyface.util
-    :members:
-    :undoc-members:
-    :show-inheritance:
+   :members:
+   :undoc-members:
+   :show-inheritance:
 
 Subpackages
 -----------
 
 .. toctree::
 
-    pyface.util.grid
-    pyface.util.tests
+   pyface.util.tests
```

Comment:
Aha apparently we have been including test cases in the API reference:
https://docs.enthought.com/pyface/api/pyface.action.tests.test_action_controller.html

Given this is an existing problem, we should open an issue for fixing this programmatically and not have this block the release.